core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerClientBuilder.java

```diff
@@ -86,6 +91,14 @@ public CircuitBreakerClientBuilder handler(CircuitBreakerClientHandler handler)
         return (CircuitBreakerClientBuilder) super.handler(handler);
     }
 
+    @Override
+    public CircuitBreakerClientBuilder recover(BiFunction<? super ClientRequestContext, ? super HttpRequest,
+            ? extends HttpResponse> fallback) {
+        requireNonNull(fallback, "fallback");
```

Comment:
How about removing this null check and adding it in `AbstractCircuitBreakerClientBuilder.recover()` instead?